keyboards/ibm/model_m_4th_gen/overnumpad_1xb/overnumpad_1xb.c

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2020 Purdea Andrei
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "overnumpad_1xb.h"
+
+void keyboard_post_init_kb(void)
+{
+    // Solenoid enable:
+    setPinOutput(C13);
+    writePin(C13, 1);
+
+    //debug_enable=true;
+    //debug_matrix=true;
+}
```

Comment:
This shouldn't be needed, as the solenoid code does this, by default.

Reply:
This is needed. Notice that this is not the solenoid trigger pin (B5). This is the solenoid enable pin (C13), which when high enables the boost converter on xwhatsit-style solenoid drivers. When this pin is left floating the xwhatsit-style solenoid driver has a pull-down on it, and it is permanently disabled.

This line will be deleted and replaced with other code, once the usb power state tracking api is added, but in the meantime it is absolutely needed.

Reply:
Since I'm at a computer I can add some extra information. See this schematic here for xwhatsit's solenoid drivers: https://static.wongcornall.com/ibm-capsense-usb-web/img/soldrv_schem.png
It has a boost converter, generating 9V and charging up C6, when the solenoid is triggered, this capacitor discharges through the solenoid. There's also some current limiting circuitry, to make sure it doesn't consume more than the USB spec allows.
This solenoid driver has two control pins: 1) the trigger (pin 4 on the connector), which is the classic solenoid pin also supported by QMK, and 2) the enable pin (pin 2 on the connector). This enable pin can be permanently driven high, while the trigger pin behaves as usual in qmk. This is exactly what this piece of code does. In the future, it will be best to turn off the enable signal, if the HPT_TOG key is pressed, or if the keyboard goes into standby in order to not stress the boost converter unnecessarily, and waste energy. But that requires some dependencies that are not yet merged, so for this PR I'm going with the basic solution of just driving it high all the time.